IFC4x3/Properties/a/AcceptanceDate_12dHQ2H3D5Vw6e1UM1sbnF/DocProperty.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="AcceptanceDate_12dHQ2H3D5Vw6e1UM1sbnF" Name="AcceptanceDate" UniqueId="429d1682-4433-457f-a1a8-05e581da5c4f" PrimaryDataType="IfcDate" />
```

Comment:
Seems like commencement date if it is modified a bit.

Reply:
nah its different i more akind to the date it is approved and accepted by the client.